llarp/router/router.cpp

```diff
@@ -139,30 +139,45 @@ namespace llarp
     // Compute all stats on all path builders on the default endpoint
     // Merge snodeSessions, remoteSessions and default into a single array
     std::vector<nlohmann::json> builders;
-    auto snode_sessions = services["default"]["snodeSessions"];
+
+    auto serviceDefault = services["default"];
+    builders.push_back(serviceDefault);
+
+    auto snode_sessions = serviceDefault["snodeSessions"];
     for (const auto& session : snode_sessions)
-      builders.push_back(session["buildStats"]);
+      builders.push_back(session);
```

Comment:
this breaks backwards compat.

Reply:
I think that's probably okay?  The only consumer of this endpoint is going to be `lokinetmon` which ought to be easily fixed.

Reply:
I took that from the old control panel https://github.com/oxen-io/lokinet-control-panel/blob/master/res/qml/ControlPanel.qml#L229

Reply:
We'll fix in another PR if it breaks lokinetmon (though Jeff thinks it might actually fix something so...)